script/build.sh

```diff
@@ -1,8 +1,10 @@
-esbuild app.jsx \
+node_modules/.bin/esbuild app.jsx \
```

Comment:
use path from `node_modules` instead of relying on the globally installed `esbuild`